### PR TITLE
Speed up online

### PR DESCRIPTION
**Pull request by agiovann**
This small changes adds a good speedup to online. I also added another way of spelling the .tif extension (.TIFF) 

Not sure what happened in terms of merging, it included some other commits from Pat, probably because I merged without committing (my bad). @pgunn can you check that everything is OK? the only real changes here are 

- added an equivalent extinsion for tif files
- speed up online by ordering in the right directions array

Let me know!

### CODE REVIEW

**Comment by pgunn** [comment]
The diff looked weird because you based it off dev but were doing the PR against master. I just adjusted it to be a PR against dev (assuming that's what you wanted).

File: caiman/base/movies.py [hunk 1]
```diff
@@ -1157,7 +1158,7 @@ def load(file_name,fr=30,start_time=0,meta_data=None,subindices=None,shape=None,
     if os.path.exists(file_name):
         _, extension = os.path.splitext(file_name)[:2]
 
-        if extension == '.tif' or extension == '.tiff':  # load avi file
+        if extension == '.tif' or extension == '.tiff' or extension == '.TIF':  # load avi file
```
Comment (pgunn) [inline caiman/base/movies.py]: We should probably just lowercase the extension right after splitting it off from the file before comparison, so we don't need to introduce case variants for all the extensions.

**Comment by agiovann** [comment]
Oops sorry. Rookie mistake. Sure.

File: caiman/base/movies.py (hunk 1 above)
Comment (epnev) [reply caiman/base/movies.py]: Agree. I looked for the appearances of the word `extension` and switched it to always be lowercase. Hopefully that should capture everything

**Pull request merged**